index.html

```diff
@@ -8,62 +8,194 @@
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			</div>
+			<button class="multiposts-btn">See Project</button>
 		  </div>
-	  </nav>
-  </header>
-  <main>
-	<section class="hero">
-	  <span class="header">
-		Hey There. I’m Alex
-	  </span>
-	  <span class="sub-header">I’am a Software Developer</span>
-	  <p class="description">
-		I can help you build a product , feature or website Look through some of my work and experience! 
-		If you like what you see and have a project you need coded, 
-		don’t hestiate to contact me.</p>
-	  <div class="social">
-		  <ul class="social-icons">
-			  <li>
-				<a href="https://github.com/Reggeditt" class="social-link">
-				  <img src="assets/images/githubIcon.png" alt="" class="icon">
-				</a>
-			  </li>
-			  <li>
-				<a href="https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/" class="social-link">
-			      <img src="assets/images/linkedin.png" alt="" class="icon">
-				</a>
-			  </li>
+          <div class="projects">
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
+			  <div >
+				<ul class="stack-list">
+					<li><a href="#">html</a></li>
+					<li><a href="#">bootstrap</a></li>
+					<li><a href="#">Ruby</a></li>
+				</ul>
+			  </div>
+			  <button class="pjt-button">See Project</button>
+			</div>
+            <div class="project-card">
+			  <span class="pjt-card-header heading">Profesional Art Printing Data</span>
+              <span class="pjt-card-sub-header sub-heading">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</span>
```

Comment:
Kindly use a `p` a paragraph tag, tThis is for correct semantics and for screen readers to understand your code.